site/docs/v1/tech/identity-providers/xbox.adoc

```diff
@@ -53,6 +53,8 @@ include::docs/v1/tech/identity-providers/_idp-form-fields.adoc[]
 
 == Building Your Own Integration
 
-:identity_provider: Xbox
-include::docs/v1/tech/identity-providers/_login_api_integration.adoc[]
-:identity_provider!:
+:identity_provider_path: Xbox
```

Comment:
should this be lowercase like the others?

Reply:
Absolutely correct. Fixed.